plugin/evm/limitorders/service.go

```diff
@@ -101,3 +107,39 @@ func (api *OrderBookAPI) GetOpenOrders(ctx context.Context, trader string) OpenO
 
 	return OpenOrdersResponse{Orders: traderOrders}
 }
+
+// NewOrderBookData send a notification each time a new (header) block is appended to the chain.
+func (api *OrderBookAPI) NewOrderBookData(ctx context.Context) (*rpc.Subscription, error) {
```

Comment:
i suggest we call it `SubscribeOrderBookState`

Reply:
when we call it we use orderbook_subscribe -> newOrderBookData(like eth_subscribe -> newHeads)
we can change it to newOrderBookState
what do you think ?